helm_deploy/values-prod.yaml

```diff
@@ -12,5 +12,11 @@ generic-service:
     EXAMPLE_API_URL: 'https://template-kotlin.hmpps.service.justice.gov.uk'
```

Comment:
Can we remove all the references to example? e.g. exampleApiClient.ts, references in config.ts, references in values files, etc.

Probably worth doing a file-search for "example" and ripping it all out...  it'll only confuse things otherwise.